packages/babel-plugin-transform-block-scoping/test/fixtures/general/issue-8498-loop-init-collision-destructuring/output.js

```diff
@@ -0,0 +1,5 @@
+for (let a, _ref = {}, b = _ref.b;;) {
+  let a,
```

Comment:
You should enable the block scoping plug-in, otherwise we are testing the wrong thing 😛

Reply:
Ahh, I thought the package plugin was enabled by default since the test is in that directory... 🤔  I'm guessing it's okay to leave the transform destructuring plugin? w/o the transform destructuring plugin, the destructured variable gets incorrectly renamed regardless.
```
var { i } = {};
{
  var { i: _i } = {};
}
```